lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -54,4 +56,19 @@ function multiplyFractions(node) {
     ChangeTypes.MULTIPLY_FRACTIONS, node, newNode);
 }
 
+// Returns true if `node` has a constant numerator and a polynomial in the denominator,
+// e.g. 5/x or 1/2x^2
+function hasConstantNumeratorAndPolynomialInDenominator(node) {
+  if (!(Node.Type.isFraction(node))) {
+    return false;
+  }
+
+  const fraction = Node.Type.getFraction(node);
+  const numerator = fraction.args[0];
+  const denominator = fraction.args[1];
+
+  return (Node.Type.isConstant(numerator) &&
```

Comment:
Do we need to ensure that the numerator is a constant? I'm not sure, I'm having trouble wrapping my head around the logic. I don't think it hurts.